bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/HeavyNeutrino_trilepton_V-0.01_tau_NLO/createCards_mNgtmWltmZ.sh

```diff
@@ -0,0 +1,24 @@
+masspoints=(90)
```

Comment:
Can you check whether 90 GeV mass point has reasonable cross section and decay BRs? NWA won't be applicable